release/release_workflow.wdl

```diff
@@ -42,133 +35,312 @@ task do_release {
 
      # Update develop to point to next release version
      git checkout develop
-     ${updateVersionCommand}
+     sed -i '' -e '/cromwellVersion[[:space:]]=/s/[0-9][0-9]*/~{nextV}/' project/Version.scala
      git add .
-     git diff-index --quiet HEAD || git commit -m "Update ${repo} version from ${releaseV} to ${nextV}"
+     git diff-index --quiet HEAD || git commit -m "Update cromwell version from ~{releaseVersion} to ~{nextV}"
      git push origin develop
+   }
+
+   output {
+     File cromwellJar = "cromwell/server/target/scala-2.12/cromwell-${releaseVersion}.jar"
+     File womtoolJar = "cromwell/womtool/target/scala-2.12/womtool-${releaseVersion}.jar"
+   }
+}
+
+# do_minor_release simply builds jars from the hotfix branch, tags the branch and push the tag to github
+task do_minor_release {
+   input {
+      # Current version being released. e.g: 33.1
+      String releaseVersion
+   
+      # Can be swapped out to try this on a fork
+      String organization
+   }
+
+   Float releaseVersionAsFloat = releaseVersion
+   Int majorReleaseNumber = floor(releaseVersionAsFloat)   
+   String hotfixBranchName = "${majorReleaseNumber}_hotfix"
+
+   command {
+     set -e
+     set -x 
 
-     pwd > executionDir.txt
+     # Clone repo and checkout hotfix branch
+     git clone git@github.com:~{organization}/cromwell.git -b ~{hotfixBranchName} cromwell
+     cd cromwell
+
+     # Make sure tests pass
+     sbt update
+     sbt test:compile
+     # Test with a backup just in case of timeouts
+     # If tests repeatedly timeout locally, ensure all tests pass in Travis and then comment out the next line
+     sbt test || sbt testQuick
+
+     # Tag the release
+     git tag ~{releaseVersion}
+
+     # Push the tags
+     git push origin ~{releaseVersion}
+
+     # Assemble jar for cromwell
+     sbt -Dproject.version=~{releaseVersion} -Dproject.isSnapshot=false assembly
    }
 
    output {
-     String version = releaseV
-     String executionDir = read_string(repo + "/executionDir.txt")
+     File cromwellJar = "cromwell/server/target/scala-2.12/cromwell-${releaseVersion}.jar"
+     File womtoolJar = "cromwell/womtool/target/scala-2.12/womtool-${releaseVersion}.jar"
    }
 }
 
 task versionPrep {
-    String organization
-    String updateCommandTemplate
-    String repo = "cromwell"
-    String file = "project/Version.scala"
+    input {
+        String organization
+        Boolean majorRelease
+    }
 
     command <<<
-      curl -o versionFile https://raw.githubusercontent.com/${organization}/${repo}/develop/${file}
-      perl -ne '/\s+cromwellVersion\s*=\s*"(.*)"/ && print $1' versionFile > currentVersion
-      echo $((`cat currentVersion` + 1)) > nextVersion
+      curl -s https://api.github.com/repos/~{organization}/cromwell/releases/latest | jq --raw-output '.tag_name'
     >>>
+    
+    runtime {
+        docker: "broadinstitute/cromwell:cromwell-release-image"
```

Comment:
Alternative to above assuming you still want to use a docker: After the [literal fires](https://twitter.com/lukwam/status/1010286524523565056) are out, let's get a real repo for this image.

I'm all for different versions/tags containing different flavors of some library/executable/artifact.

But this is like someone going to download https://github.com/broadinstitute/cromwell/releases/download/kshakir-special/cromwell-kshakir-special.jar and discovering the jar only contains Minecraft.

Reply:
>  let's get a real repo for this image.
Agreed, I think it'd be nice to have this as portable as possible and therefore use docker images (eventually maybe even for all the tasks). But until we can get a proper repo we can do what you suggested below and install jq on the machine directly.